kubetest2-aks/deployer/up.go

```diff
@@ -230,6 +230,8 @@ func (d *deployer) prepareClusterConfig(clusterID string) (*armcontainerservicev
 		return nil, "", fmt.Errorf("failed to prepare custom config: %v", err)
 	}
 
+	klog.Infof("Customized configurations are: %s", string(customConfig))
```

Comment:
the encoded custom configuration as a string in custom config can be truncated, so print the custom configuration explicitly for better debugging.